agent/engine/task_manager.go

```diff
@@ -358,16 +366,17 @@ func (mtask *managedTask) handleContainerChange(containerChange dockerContainerC
 		now := mtask.time().Now()
 		ok := mtask.Task.SetExecutionStoppedAt(now)
 		if ok {
-			seelog.Infof("Recording execution stopped time for a task, essential container in task stopped, task %s, time: %s",
-				mtask.Task.String(), now.String())
+			seelog.Infof("Managed task [%s]: recording execution stopped time. Essential container %s stopped at: %s",
```

Comment:
nit: `container [%s]`, I think there are few of these where formatting varies from line to line. Not sure if this was intentional.